service/scaladsl/server/src/main/resources/reference.conf

```diff
@@ -0,0 +1,10 @@
+lagom.logging {
+
+  default = "WARN"
+
+  exception.logging.level = {
+    "com.lightbend.lagom.scaladsl.api.transport.BadRequest" = "WARN"
+    "com.lightbend.lagom.scaladsl.api.transport.Forbidden" = "ERROR"
+    "com.lightbend.lagom.scaladsl.api.transport.NotFound" = "DEBUG"
+  }
+}
```

Comment:
I'd prefer sticking to the current default logs on this PR and discuss new defaults in a different PR.

```
lagom.logging {

  default = "ERROR"

  exception.logging.level = {
    "com.lightbend.lagom.scaladsl.api.transport.UnsupportedMediaType" = "WARN"
    "com.lightbend.lagom.scaladsl.api.transport.PayloadTooLarge" = "WARN"
    "com.lightbend.lagom.scaladsl.api.transport.NotAcceptable" = "WARN"
    "com.lightbend.lagom.scaladsl.api.transport.NotFound" = "TRACE"
    "com.lightbend.lagom.scaladsl.api.transport.Forbidden" = "TRACE"
  }
}
```

PS: I am aware that the PR is currently a PoC. :-)